src/classes/Product.php

```diff
@@ -49,6 +49,11 @@ public function getAvailableColors()
         return $this->availableColors;
     }
 
+    public function getImageFilePath()
+    {
+        return $this->imageFilePath;
```

Comment:
Are you aware that the product can have multiple images & that this may return an array?